backend/main.py

```
@@ -1,241 +1,380 @@
+
+# --------------------------------------------------------------------------- #
+# Instantiate global service singletons
+# --------------------------------------------------------------------------- #
+coach_service = CoachService()
+persona_service = PersonaService()
+tts_service = TtsService()
+
+# Create necessary directories using settings
+settings.UPLOAD_DIR.mkdir(parents=True, exist_ok=True)
+settings.TEMP_DIR.mkdir(parents=True, exist_ok=True)
+settings.EXPORT_DIR.mkdir(parents=True, exist_ok=True) # Ensure export directory exists
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)